src/CollAction/Models/ProjectViewModels/CreateProjectViewModel.cs

```diff
@@ -58,5 +60,13 @@ public CreateProjectViewModel(SelectList categories, SelectList locations)
         [Required]
         [DisplayFormat(DataFormatString = "{0:dd/MM/yyyy}", ApplyFormatInEditMode = true)]
         public DateTime End { get; set; }
+
+        [Display(Name = "Banner image", Prompt = "1366x767px JPEG, GIF, PNG, BMP")]
+        [FileSize(1024000)] // 1MB
+        [FileType("jpg, jpeg, gif, png, bmp")]
+        [MinImageDimensions("1366x768")]
```

Comment:
As per CA-170 "For the header: maximum 1024 x 768 op 72 dpi" I will be changing the MinImageDimensionsAttribute class to MaxImageDimensions("1024x768").